app/src/androidTest/java/com/example/roomremix2/ExampleInstrumentedTest.kt

```diff
@@ -21,4 +27,27 @@ class ExampleInstrumentedTest {
         val appContext = InstrumentationRegistry.getInstrumentation().targetContext
         assertEquals("com.example.roomremix2", appContext.packageName)
     }
+}
+class MainActivityTest {
+    @Test
+    fun onClick_drawButtonBlack() {
+        ActivityScenario.launch(MainActivity::class.java)
+        onView(withId(draw)).perform(click())
+        assert(DrawView.currentColor == Color.BLACK)
+    }
+    @Test
+    fun onClick_eraserButtonWhite(){
+        ActivityScenario.launch(MainActivity::class.java)
+        onView(withId(eraser)).perform(click())
+        assert(DrawView.currentColor == Color.WHITE)
+    }
+    @Test
+    fun onClick_deleteButtonWorks(){
+        ActivityScenario.launch(MainActivity::class.java)
+        DrawView.pathList.add(Path())
+        DrawView.colorList.add(Color.BLACK)
+        onView(withId(delete)).perform(click())
+        assert(DrawView.pathList.isEmpty())
+        assert(DrawView.colorList.isEmpty())
+    }
```

Comment:
Better unit test coverage is always a good idea.

Be cautious with UI tests, as they tend to be brittle.  One small UI change can invalidate an entire series of tests.